src/sentry/integrations/vercel/integration.py

```diff
@@ -255,12 +255,14 @@ def update_organization_config(self, data):
                 "SENTRY_PROJECT_%s" % uuid,
                 dsn_secret_name % uuid,
                 "SENTRY_AUTH_TOKEN_%s" % uuid,
+                f"VERCEL_{source_code_provider.upper()}_COMMIT_SHA_{uuid}",
```

Comment:
@ceorourke Is this due to a change in the V6 endpoint?

Reply:
@scefali yeah I was kinda confused why we weren't creating a secret for this before, or rather we were passing it as an empty string. After upgrading the endpoint version I was hitting an error doing that so I had to create one.